src/main/environment/common_ci.properties

```diff
@@ -0,0 +1,89 @@
+# CI environment for building war files
+# DB Connections 
+spring.datasource.url=@env.DATABASE_URL@
+spring.datasource.username=@env.DATABASE_USERNAME@
+spring.datasource.password=@env.DATABASE_PASSWORD@
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+
+## Carestream URLs
+carestreamOrderCreateURL =@env.COMMON_API_BASE_URL@/carestream/createOrder
+
+## Identity - Common URLs
+#  Registration
+registrationUrl =@env.COMMON_API_BASE_URL@/beneficiary/create
+
+syncSearchByLocation =@env.COMMON_API_BASE_URL@/id/searchByDistrictId
+
+registrarQuickSearchByIdUrl =@env.COMMON_API_BASE_URL@/beneficiary/searchUserByID
+
+registrarQuickSearchByPhoneNoUrl =@env.COMMON_API_BASE_URL@/beneficiary/searchUserByPhone
+
+getBenImageFromIdentity =@env.COMMON_API_BASE_URL@/id/benImageByBenRegID
+
+##save covid Vaccine details coming from mobile app through fhir
+saveCovidVaccineDetailsURL = @env.COMMON_API_BASE_URL@/covid/saveCovidVaccinationDetails
+
+##beneficiary edit
+beneficiaryEditUrl =@env.COMMON_API_BASE_URL@/beneficiary/update
+
+## Advance Search
+registrarAdvanceSearchUrl =@env.COMMON_API_BASE_URL@/beneficiary/searchBeneficiary
+
+## Data Sync API
+dataSyncUploadUrl=@env.MMU_API_BASE_URL@/dataSync/van-to-server
+
+## Data download API
+dataSyncDownloadUrl=@env.MMU_API_BASE_URL@/dataSync/server-to-van
+
+## TC specialist slot booking
+tcSpecialistSlotBook=@env.SCHEDULER_API_BASE_URL@/schedule/bookSlot
+
+## TC specialist slot cancelling
+tcSpecialistSlotCancel=@env.SCHEDULER_API_BASE_URL@/schedule/cancelBookedSlot
+
+## TM sms to beneficiary(schedule, cancel, reschedule)
+sendSMSUrl=@env.COMMON_API_BASE_URL@/sms/sendSMS
+
+### get openkm doc download url
+openkmDocUrl=@env.COMMON_API_BASE_URL@/kmfilemanager/getKMFileDownloadURL
+
+##Fetosense Url
+foetalMonitor-api-url-ANCTestDetails=https://asia-south1-amrit-fetosense.cloudfunctions.net/insertMother
+
+## apiKey for calling fetosense api
+foetalMonitorAPIKey = @env.FETOSENSE_API_KEY@
+
+## TM SMS template details(SMS type)
+schedule=TC Schedule SMS
+cancel=TC Cancel SMS
+reSchedule=TC Reschedule SMS
+
+snomedCTPageSize=10
+
+nurseWL=7
```

Comment:
@Anjana2310  do you know what these IDs are?
Do they change as per the environment?
If yes, I will mark it as an env variable.

Reply:
@drtechie These variables do not vary with environment. snomedCTPageSize=10
 refer to the size of the page to be returned and nurseWL and other WL variables are realted to Worklist implementation which is consistent across all environments.